### ISSUE TITLE
CNN + SVM

### ISSUE THREAD

**Issue by hdriqi** [apache/incubator-mxnet#6574]
So I am trying to create a CNN + SVM model for CIFAR-10 dataset

```
data = mx.sym.var('data')

conv1 = mx.sym.Convolution(data=data, kernel=(5,5), num_filter=20)
tanh1 = mx.sym.Activation(data=conv1, act_type="tanh")
pool1 = mx.sym.Pooling(data=tanh1, pool_type="max", kernel=(2,2), stride=(2,2))

conv2 = mx.sym.Convolution(data=pool1, kernel=(5,5), num_filter=50)
tanh2 = mx.sym.Activation(data=conv2, act_type="tanh")
pool2 = mx.sym.Pooling(data=tanh2, pool_type="max", kernel=(2,2), stride=(2,2))

flatten = mx.sym.flatten(data=pool2)
fc1 = mx.symbol.FullyConnected(data=flatten, num_hidden=500)
tanh3 = mx.sym.Activation(data=fc1, act_type="tanh")

fc2 = mx.sym.FullyConnected(data=tanh3, num_hidden=10)

svm = mx.sym.SVMOutput(data=fc2, name='svm')

model = mx.mod.Module(symbol=svm, context=mx.cpu(), label_names=['svm_label'])
model.fit(train_iter,
		eval_data=val_iter,
		optimizer='adam',
		optimizer_params={'learning_rate':0.1},
		batch_end_callback = mx.callback.Speedometer(batch_size, 500),
		num_epoch=100)  

test_iter = mx.io.NDArrayIter(images_test, labels_test, batch_size)
acc = mx.metric.Accuracy()
model.score(test_iter, acc)
print(acc)
```

There's no error while running the code, but the train-accuracy and validation-accuracy do not increase although it is already run on 50 epoch ( from 1st epoch till 50 still around 0.1 ).

Is there anything that I miss for using the SVMOutput? because when I use SoftmaxOutput the accuracy is increasing for every epoch.

**Comment by ysh329** [comment]
Can you give the full log?

**Comment by yajiedesign** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
